src/01-general/custom-fields.apib

```diff
@@ -0,0 +1,131 @@
+## Custom Fields [/customFieldDefinitions]
+
+### customFieldDefinitions.list [GET /customFieldDefinitions.list]
+
+Get a list of all the definitions of custom fields.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + filter (object, optional)
+            + ids: `74855f4a-2b61-429c-81d8-c79ad3675a76`,`1a68ac98-b41f-4c60-af12-895c43771385` (array[string], optional)
+            + context: `contact` (enum[string], optional) - The context of the custom field definition
```

Comment:
Not sure about name "context", but don't really have a better idea either.

full list:
'contact','company','form','event','product','offertex_lead_category_link','project','sale','invoice','meeting_report','subscription','werkbonnen','inv_invoice_item','ticket','milestone','project_thirdparty','meeting','callback','todo','pro_external_cost'

Reply:
`entity`, `object`, `on`, `owner`, ... :)

Reply:
let's only focus on what we want to release right now ...